docs-md/concepts/materialized-views.md

```diff
@@ -1,16 +1,53 @@
 ---
 layout: page
-keywords: ksqldb, query, select
+title: Materialized Views
+tagline: Aggregation over streams in ksqlDB
+description: Learn about aggregation and materialized views in ksqlDB.  
+keywords: ksqldb, query, select, aggregate, materialized view
 ---
 
-One of the main purposes of tables in any database is to enable efficient queries over the data. We’ve seen how ksqlDB stores events immutability in Kafka with a simple key/value model. How can queries be made efficient under this model? The answer is by leveraging materialized views. How this works requires a bit of an explanation.
+In any database, one of the main purposes of a table is to enable efficient
+queries over the data. ksqlDB stores events immutably in {{ site.aktm }} by
+using a simple key/value model. But how can queries be made efficient under
+this model? The answer is by leveraging materialized views.
 
-Streams and tables share an intimate relationship. A stream is a sequence of events from which a table can be derived. For example, a sequence of credit scores for a loan applicant can change over time. The sequence of credit scores is a stream. But that stream be interpreted as a table to describe the applicant’s current credit score. On the flip side, the table that represents current credit stores is really two things: the current credit scores, and also the sequence of changes to the credit scores for each applicant. This is a profound realization, and much has been written on this stream/table duality.  Many databases throw away the series of changes to a table and only keep the current state around, thus breaking the natural duality.
+Streams and tables are closely related. A stream is a sequence of events that
+you can derive a table from. For example, a sequence of credit scores for a
+loan applicant can change over time. The sequence of credit scores is a stream.
+But this stream be interpreted as a table to describe the applicant's current
+credit score.
 
-For our purposes, the important thing that enables efficient queries is access to the current state of applied changes. This is more commonly known as a materialized view. 
+Conversely, the table that represents current credit stores is really two
+things: the current credit scores, and also the sequence of changes to the
+credit scores for each applicant. This is a profound realization, and much has
+been written on this stream/table duality.  Many databases throw away the
+series of changes to a table and only keep the current state around, which

```

Comment:
Most databases do keep Redo logs around right?

Reply:
Right, I was trying to say that most DB's don't let you access the history of a field if you mutate it in place. Can you help me say that a little clearer?

Reply:
Actually they do. Redo logs just have much shorter retention than your kafka changelog topic. A fully compacted kafka changelog topic is same as a db snapshot. 

I think we can talk to how cumbersome it is in databases to subscribe to changes. Thats the real win here IMO. I ll let you decide :)

Reply:
That's a much better way to say what I was trying to get at. Nice.